src/scenes/orders/dropdownOrders.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from "react";
+import styles from "./dropdowndOrders.module.scss";
+import Select, { GroupBase, SingleValue, StylesConfig } from "react-select";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  currentOrderIndexSelector,
+  ordersSelector,
+} from "../../store/orders/selectors";
+import {
+  updateCurrentOrderId,
+  updateCurrentOrderIndex,
+} from "../../store/orders/ordersSlice";
+
+export interface DropdownInfo {
+  value: number;
+  label: string;
+}
+
+export function DropdownOrders({}) {
+  const dispatch = useDispatch();
+  const orders = useSelector(ordersSelector);
+  const currentOrderIndex = useSelector(currentOrderIndexSelector);
+
+  const dropdownOptions: DropdownInfo[] = orders.map((order) => {
+    return { value: order.id, label: order.name };
+  });
+
+  const handleChange = (selectedOption: SingleValue<DropdownInfo>) => {
+    if (selectedOption !== null) {
+      const currentOrder = orders.find(
```

Comment:
maybe use straight away findIndex?

Reply:
I didn't know that method ;) It simplifies a lot. Thank you!